### PR TITLE
Add test for ob issued eidas certificate

### PR DESCRIPTION
**Pull request by benjefferies**
A TPP has struggled to onboard with OB issue eidas certificates so
adding a failed test with his public cert.

The reason for the failure is that it doesn't have the `0.4.0.1862.1.1` qc statement. Any ideas @BohoCode ?

### CODE REVIEW

**Comment by BohoCode** [comment]
Also includes a fix for #6

File: src/test/java/com/forgerock/cert/Psd2CertInfoTest.java [hunk 1]
```diff
@@ -85,13 +84,18 @@ public void canReadAuthorityAccessInfo() throws IOException, CertificateExceptio
...
+     */
+    @Ignore
     public void getOrganisationId() throws NoSuchRDNInField, CertificateException, IOException {
-        String expectedOrgId = "PSDGB-" + CertificateUtils.getRDNAsString(this.certs[0], RdnField.SUBJECT,
-                BCStyle.OU);
         Optional<String> orgIdOptional = this.psd2CertInfo.getOrganizationId();
         assertThat(orgIdOptional.isPresent(), is(this.testSpec.isPsd2Cert()));
         if(this.testSpec.isPsd2Cert()) {
-            assertThat(orgIdOptional.get(), is(expectedOrgId));
+            assertThat(orgIdOptional.get(), RegexMatcher.matches("PSD[A-Z]{2}-[A-Z]{2,8}-.*"));
```
Comment (benjefferies) [inline src/test/java/com/forgerock/cert/Psd2CertInfoTest.java]: Maybe you could have used http://hamcrest.org/JavaHamcrest/javadoc/2.0.0.0/org/hamcrest/text/MatchesPattern.html

Comment (BohoCode) [reply src/test/java/com/forgerock/cert/Psd2CertInfoTest.java]: Would have meant bumping version which brought other challenges

**Comment by BohoCode** [comment]
> Also includes a fix for #6

Ignore this!

**Pull request merged**